transit_odp/timetables/tests/test_csv.py

```diff
@@ -42,18 +42,16 @@ def test_service_in_bods_but_not_in_otc():
         assert row["Published Status"] == "Published"
         assert row["OTC Status"] == "Unregistered"
         assert row["Scope Status"] == "In Scope"
-        assert row["DQ Score"] == "33%"
```

Comment:
Not on this line but there are a few test cases that are failing

Reply:
Fixed this @alvak1